partner_create_by_vat/models/res_partner.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# See README.rst file on addons root folder for license details
+
+import requests
+
+from string import maketrans
+from stdnum.eu.vat import check_vies
```

Comment:
You still need the stdnum, because if it's not a romanian company, it will use the vies service, and by default vatnumber from python doesn't retrieve the fields: name, adress from vies like stdnum does...